spec/index.bs

```diff
@@ -3910,7 +3910,8 @@ enum GPUTextureDimension {
     : <dfn>"3d"</dfn>
     ::
         Specifies a texture that has a width, height, and depth. {{GPUTextureDimension/"3d"}}
-        textures cannot be multisampled or use depth/stencil formats.
+        textures cannot be multisampled, and their format must support 3d textures
+        (all [[#plain-color-formats]] and some [[#packed-formats|packed/compressed formats]]).
```

Comment:
My bad, I realized this will render poorly without a text override.
> (all § 26.1.1 Plain color formats and some packed/compressed formats)

```suggestion
        (all [[#plain-color-formats|plain color formats]] and some [[#packed-formats|packed/compressed formats]]).
```